api_pos_integration.md

```diff
@@ -0,0 +1,277 @@
+| `netPrice`     | `int`          |         | The net price                                                                                                                                  |
+| `price`        | `int`          |         | Total price                                                                                                                                    |
+| `state`        | `PaymentState` |         | The [Status of the payment process](api_checkout.md#payment-state). Only `pending`, `processing`, `successful`, `failed` are used in this case |
+| `loyaltyCard`  | `String`       |         | The loyalty card identifier provided by the user                                                                                               |
+| `processedBy`  | `String`       |         | String identifying the PoS which processes the checkout process                                                                                |
+| `failedReason` | `String`       |         | Message describing the conditions under which the checkout process was marked as failed                                                        |
+
+#### Line Item
+
+| Parameter     | Type     | Default | Description                                                       |
+|---------------|----------|---------|-------------------------------------------------------------------|
+| `scannedCode` | `string` |         | Scanned code                                                      |
+| `amount`      | `int`    |         | Number of products / packages                                     |
+| `weight`      | `int`    | 0       | Weight of product in case of a weighable (a not packaged) product |
+| `units`       | `int`    | 0       | Number of units in a package in case of bundle or piece product   |
+| `price`       | `int`    |         | The base price of one product / unit                              |
+| `totalPrice`  | `int`    |         | The total price                                                   |
+| `taxRate`     | `string` |         | Tax rate as string encoded decimal                                |
+
+#### Example
+```
+{
+  "lineItems" : [
+    {
+      "scannedCode": "0000000000001",
+      "totalPrice" : 798,
+      "amount" : 2,
+      "price" : 399,
+      "taxRate" : 19,
+      "sku" : "1"
+    },
+    {
+      "scannedCode": "0000000000002"
+      "price" : 1099,
+      "totalPrice" : 1099,
+      "amount" : 1,
+      "taxRate" : 19
+    }
+  ],
+  "tax" : {
+    "19" : 7673
+  },
+  "taxNet" : {
+    "19" : 40382
+  },
+  "netPrice" : 40382,
+  "price" : 48055,
+  "state": "pending",
+  "loyaltyCard": "12345678",
+  "processedBy": "pos-1"
+}
+```
+
+## Get Checkout Process
+`GET /{project}/pos/checkout/id/{id}`
+
+Get the state of a Checkout Process.
+
+**Required permissions** : checkoutRead for the checkout process
```

Comment:
checkoutRead for the project 
Wir haben ja keine process basierten Rechte sondern immer noch Projekt basierte oder?